torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1252,15 +1252,49 @@ Error PyTorchModelLoader::loadGlowFusedLinear(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], output);
 }
 
+Expected<NodeValue> PyTorchModelLoader::loadNodeValueOrBroadcastedConstant(
+    const torch::jit::Value *value, llvm::ArrayRef<size_t> dims) {
+  if (hasGlowNodeValueForValue(value)) {
+    return getGlowNodeValueForValue(value);
+  } else {
+    GlowIValue *ival;
+    ASSIGN_VALUE_OR_RETURN_ERR(ival, getGlowIValueForValue(value));
+    float constVal;
+    if (ival->isInt()) {
```

Comment:
Basically we cast them to float in Glow?

Reply:
Turns int into float tensor because we can't add ints in Glow